### PR TITLE
Added new option to compareECL 

### PR DESCRIPTION
**Pull request by tskille**
-d : Use report steps only when comparing results from summary files.

### CODE REVIEW

**Comment by bska** [comment]
jenkins build this please

File: src/opm/io/eclipse/ESmry.cpp [hunk 1]
```diff
@@ -58,7 +58,10 @@ ESmry::ESmry(const std::string &filename, bool loadBaseRunData)
     bool formatted=false;
     
     char buff[1000];
-    getcwd( buff, 1000 );
+    
+    if (getcwd( buff, 1000 )==NULL){
```
Comment (joakim-hove) [inline src/opm/io/eclipse/ESmry.cpp]: When you use fixed length buffers you could use `PATH_MAX` which is defined in header `limits.h`

File: opm/io/eclipse/ESmry.hpp [hunk 2]
```diff
@@ -34,6 +34,9 @@ class ESmry
     bool hasKey(const std::string& key) const;
 
     const std::vector<float>& get(const std::string& name) const;
+
+    std::vector<float> get_rstep(const std::string& name) const;
```
Comment (joakim-hove) [inline opm/io/eclipse/ESmry.hpp]: Maybe call it `get_at_rstep()` or something similar - the current name seems like you are looking for the report steps themselves.

File: CMakeLists_files.cmake [hunk 3]
```diff
@@ -335,6 +336,7 @@ if(ENABLE_ECL_INPUT)
     tests/SPE1_TESTCASE.F0025
     tests/SPE1_TESTCASE.X0025
     tests/SPE1CASE1.UNSMRY
+    tests/SPE1CASE1A.UNSMRY
```
Comment (joakim-hove) [inline CMakeLists_files.cmake]: Is it necessary to add a new binary summary case - could you not use the already existing `SPECASE1`? Or alternatively switch to let all tests use the new?

File: opm/io/eclipse/ESmry.hpp (hunk 2 above)
Comment (tskille) [reply opm/io/eclipse/ESmry.hpp]: agree, get_at_rstep is better. Changed

File: src/opm/io/eclipse/ESmry.cpp (hunk 1 above)
Comment (tskille) [reply src/opm/io/eclipse/ESmry.cpp]: fixed

**Comment by tskille** [comment]
If not more comments, ready be be merged from my side

**Comment by bska** [comment]
jenkins build this please

**Pull request merged**